docs/design.html

```diff
@@ -254,32 +254,32 @@ <h3 class="anchor-heading"><a id="semantics" class="anchor-link"></a><a href="#s
         <i>At least once</i>&mdash;Messages are never lost but may be redelivered.
     </li>
     <li>
-        <i>Exactly once</i>&mdash;this is what people actually want, each message is delivered once and only once.
+        <i>Exactly once</i>&mdash;Each message is delivered once and only once.
```

Comment:
We should not say "delivered" (also further above) -- this term is too close to "delivery semantics at the network layer", -- we still retry internally, and _deliver_ a message more than once (holds true for idempotent producer, as well as aborted TX which are retried, and aborted stuff is just filtered later).

Btw: Further above we says:
> let's discuss the semantic guarantees Kafka provides between producer and consumer

This is also misleading, because EOS works for read-process-write, but not for upstream producer to downstream consumer. While the consumer is guarded to not return aborted messages to the app, it's still at-least-once as there is not guarantee that a committed messages is only read exactly-once.

Reply:
I will try for "processed" then.